includes/class-wc-amazon-payments-advanced-api.php

```diff
@@ -1959,4 +1961,35 @@ public static function send_email_notification( $subject, $message, $recipient )
 		$mailer->send( $recipient, strip_tags( $subject ), $message );
 	}
 
+	/**
+	 * Return if subscription is shippable
+	 *
+	 * @param WC_Order $order Order object.
+	 *
+	 * @return boolean
+	 */
+	public static function maybe_subscription_is_shippable( WC_Order $order ) {
+
+		if ( ! class_exists( 'WC_Subscriptions_Product' ) ) {
+			return;
+		}
```

Comment:
As in case below this one should return explicit `false`.
```suggestion
		if ( ! class_exists( 'WC_Subscriptions_Product' ) ) {
			return false;
		}
```